### DISCUSSION TITLE
no code generate when i create relationship post model to category

### DISCUSSION THREAD

**Discussion by madfortech** [misterdebug/crud-generator-laravel#44] [Q&A] [answered]
![Screenshot 04-13-2024 13 18 20](https://github.com/misterdebug/crud-generator-laravel/assets/21261716/bf0ec106-c2db-4c47-aa9f-07d40f0aaebc)
 I will generate two crud first postcontroller and asking relationship will set category 
 second crud category controller and asking relationship i will set post
 but when i see post or category model there is no code 
 here is final output
 
![Screenshot 04-13-2024 13 21 14](https://github.com/misterdebug/crud-generator-laravel/assets/21261716/d2c551b2-3e51-4ee0-8b94-b29fdefa8880)  
same as post model 
sorry my english weak help me

**Comment by misterdebug** [comment, marked as answer]
If you want add relationships you must to reply yes not post
